config/database.yml

```diff
@@ -1,6 +1,8 @@
 development:
   adapter: postgresql
-  database: discourse_development
+  database: <%= ENV['DISCOURSE_DB_NAME'] || 'discourse_development' %>
```

Comment:
The current arrangement is: when running with Rails.env.production, use environment variables. Otherwise, use database.yml. Are you suggesting I change that?